core/renderers/block_rendering_rewrite/block_render_info.js

```diff
@@ -103,6 +109,12 @@ Blockly.blockRendering.RenderInfo = function(block) {
    */
   this.rows = [];
 
+  /**
+   * An array of measureable objects containing hidden icons.
+   * @type {Array}
```

Comment:
Make this more specific:
`@type {!Array.<!Blockly.blockRendering.Icon>}`

Reply:
Done